src/app/shared/orderby.pipe.ts

```diff
@@ -0,0 +1,20 @@
+//Import from '@angular/core' the module Pipe and PipeTransform
+import { Pipe, PipeTransform } from '@angular/core';
+
+//Tell Angular2 we're creating a Pipe with TypeScript decorators
+@Pipe({
+	name: 'orderby'
+})
+export class OrderByPipe implements PipeTransform {
+	transform(courses: any[], orderBy: string) {
```

Comment:
the pipe is intended to be universal and domain-agnostic. Try omitting business-related variable names such as `courses` in pipes

Reply:
Agree. I defined the array as `any []` for this purpose, but the name is bad.